apps/meteor/client/views/marketplace/AppMenu.js

```diff
@@ -40,6 +41,9 @@ function AppMenu({ app, isAppDetailsPage, ...props }) {
 	const buildExternalUrl = useEndpoint('GET', '/apps');
 	const syncApp = useEndpoint('POST', `/apps/${app.id}/sync`);
 	const uninstallApp = useEndpoint('DELETE', `/apps/${app.id}`);
+	const { data } = useIsEnterprise();
+
+	const isEnterpriseLicense = data?.isEnterprise ?? false;
```

Comment:
is it possible for data.isEnterprise to always be a boolean? so we can avoid the increase in complexity by adding ?? false here.

Reply:
Since the data can be undefined, I am just guaranteeing that this variable is at least false

Reply:
There is another option by casting using boolean cast `!!data?.isEnterprise`